docs/react/reference/useQueries.md

```diff
@@ -25,4 +25,4 @@ The `useQueries` hook accepts an options object with a **queries** key whose val
 
 **Returns**
 
-The `useQueries` hook returns an array with all the query results.
+The `useQueries` hook returns an ordered array with all the query results.
```

Comment:
order how? can we elaborate on that?

Reply:
I have fixed it with this, hopefully it's a bit more clear:

The `useQueries` hook returns an array with all the query results. The order returned is the same as the input order.